### PR TITLE
Add sanger dashboard

### PR DESCRIPTION
**Pull request by moonso**

### CODE REVIEW

File: scout/server/blueprints/dashboard/views.py [hunk 1]
```diff
@@ -36,35 +36,37 @@ def index():
...
-        if doc_id in var_ids:
-            continue
-        var_ids.add(doc_id)
-
-        validation = variant_obj.get('validation', 'not_evaluated')
-
-        # Check that the variant is not evaluated
-        if validation in ['True positive', 'False positive']:
-            nr_evaluated += 1
+        # variant_id = event['variant_id']
```
Comment (dnil) [inline scout/server/blueprints/dashboard/views.py]: # Ahem? #

Comment (moonso) [reply scout/server/blueprints/dashboard/views.py]: Hmm this code does not correspond to my local...

**Comment by moonso** [comment]
It should be correct now @dnil

**Comment by moonso** [comment]
So @dnil check this out 🚀

File: scout/server/blueprints/dashboard/controllers.py [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    if nr_ordered:
+        variants = [
+            {
+                'title': 'Validation ordered',
+                'count': nr_ordered
+            },
+            {
+                'title': 'Validated True Positive',
+                'count': len(validated_tp),
+                'percent': len(validated_tp) / nr_ordered,
```
Comment (dnil) [inline scout/server/blueprints/dashboard/controllers.py]: I still think this should be tp / (tp + fp), in particular as not all clinics (*ahem* cust002 *ahem*) use the verification email extensively yet.

File: scout/server/blueprints/dashboard/controllers.py [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+            },
+            {
+                'title': 'Validated False Positive',
+                'count': len(validated_fp),
+                'percent': len(validated_fp) / nr_ordered,
```
Comment (dnil) [inline scout/server/blueprints/dashboard/controllers.py]: ..and the same here obviously.

File: scout/constants/indexes.py [hunk 4]
```diff
@@ -51,16 +51,6 @@
             name="caseid_variantid",
             background=True,
             ),
-        IndexModel([
```
[the remark this answers is not in the document]
Comment (moonso) [reply scout/constants/indexes.py]: Just wanted to remove that when I looked at the file...

File: scout/server/blueprints/dashboard/controllers.py (hunk 2 above)
Comment (moonso) [reply scout/server/blueprints/dashboard/controllers.py]: Sure I'll fix that

**Pull request merged**